src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -305,18 +443,28 @@ protected override ImmutableArray<PendingBranch> Scan(ref bool badRegion)
             MethodSymbol methodSignatureOpt,
             ArrayBuilder<(BoundReturnStatement, TypeSymbolWithAnnotations)> returnTypes,
             VariableState initialState,
-            Action<BoundExpression, TypeSymbolWithAnnotations> callbackOpt)
+            PooledDictionary<BoundExpression, (NullabilityInfo, TypeSymbol)> analyzedNullabilityMapOpt = null)
```

Comment:
>PooledDictionary [](start = 12, length = 16)

`Dictionary<,>`

Reply:
I don't understand this.

---
In reply to: [265689270](https://github.com/dotnet/roslyn/pull/34017#discussion_r265689270) [](ancestors = 265689270)